aiortc/rtcpeerconnection.py

```diff
@@ -37,6 +37,12 @@ def find_common_codecs(local_codecs, remote_codecs):
     for c in remote_codecs:
         for codec in local_codecs:
             if codec.name == c.name and codec.clockRate == c.clockRate:
+                if codec.name == 'H264':
+                    # FIXME: check according to rfc 3184
+                    if c.parameters.get('profile-level-id') != '42001f':
```

Comment:
Instead of hard-coding values, why not compare to codec.parameters?

Reply:
I just hard-code to move forward. But I think it must be generalized, because compare parameters compatibility is codec specific, but I not figure out how to do

Reply:
OK that's fine for now. Do make sure you update the `RTCRtpCodecParametersTest` unit test to cover these new code branches though.